docs/api/access-keys.md

```diff
@@ -3,16 +3,19 @@ id: access-keys
 title: Access Keys Management APIs
 sidebar_label: Access Keys
 description: Use HTTP endpoints to manage access keys.
-hide_table_of_contents: true
+hide_table_of_contents: false
```

Comment:
`false` is the default, so you can just remove that `hide_table_of_contents: true` line.

```suggestion
```

Reply:
Thanks, @kimsauce. Fixed.